tests/benchmarking/tst_benchmarking_inprocess.qml

```diff
@@ -0,0 +1,123 @@
+/**********************************************************************
+**
+** Copyright (C) 2019 Luxoft Sweden AB
+**
+** This file is part of the FaceLift project
+**
+** Permission is hereby granted, free of charge, to any person
+** obtaining a copy of this software and associated documentation files
+** (the "Software"), to deal in the Software without restriction,
+** including without limitation the rights to use, copy, modify, merge,
+** publish, distribute, sublicense, and/or sell copies of the Software,
+** and to permit persons to whom the Software is furnished to do so,
+** subject to the following conditions:
+**
+** The above copyright notice and this permission notice shall be
+** included in all copies or substantial portions of the Software.
+**
+** THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+** EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+** MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+** NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+** BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+** ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+** CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+** SOFTWARE.
+**
+** SPDX-License-Identifier: MIT
+**
+**********************************************************************/
+
+import QtTest 1.2
+import tests.benchmarking 1.0
+import "BenchmarkingHelper.js" as Benchmark
+
+TestCase {
+    name: "benchmark-models-inprocess"
+
+    BenchMarkingInterface {
+        IPC.enabled: true
+        IPC.objectPath: "/benchmark/models/inprocess"
+    }
+
+    BenchMarkingIPCProxy {
+        id: api
+        ipc.objectPath: "/benchmark/models/inprocess"
+    }
+
+
+    function initTestCase() {
+        Benchmark.checkInit();
```

Comment:
BenchMark, Benchmark, Benchmarking ?
Please make up your mind and be consistent